src/usr/local/www/vpn_ipsec_keys_edit.php

```diff
@@ -186,9 +192,15 @@ function build_ipsecid_list() {
 
 $section->addInput(new Form_IpAddress(
 	'pool_address',
-	'*IPv4 address',
+	'*Virtual Address Pool',
```

Comment:
Why the name change? If it must be an IPv4 Address, the original name seems fitting.

Also if it's optional we can remove that `*` from before the name since that denotes a required option. While you are here, the option just above, "Identifier Type", can also have the `*` removed since it is optional.

Reply:
Virtual Address Pool seems to fit better, because you can choose the subnet yourself. For me, IPv4 Address means one single host...

for the rest, you are absolutely right. will change it.

Reply:
But it still only assigns the client a single address with the chosen subnet mask, right?
I suppose the description works either way.

Reply:
yes, every client will get its own address based on the subnet. It's the same like the Virtual Address Pool in "VPN/IPsec/Mobile Clients". Now the description is consistent over different (IPsec) settings pages.